sklearn/ensemble/_stacking.py

```diff
@@ -197,6 +197,15 @@ def fit(self, X, y, sample_weight=None):
 
         return self
 
+    @property
```

Comment:
Could be worth using the same structure as others meta estimators for consistency

```py
    def n_features_in_(self):
        # For consistency with other estimators we raise a AttributeError so
        # that hasattr() fails if the estimator isn't fitted.
        try:
            check_is_fitted(self)
        except NotFittedError as nfe:
            raise AttributeError(
                "{} object has no n_features_in_ attribute."
                .format(self.__class__.__name__)
            ) from nfe

        return self.estimators_[0].n_features_in_
```

Reply:
Not for this PR but thinking about it, would it make sense to have something shared in a base class (or mixin).